regression/strings-smoke-tests/CMakeLists.txt

```diff
@@ -1,3 +1,10 @@
 add_test_pl_tests(
     "$<TARGET_FILE:jbmc>"
 )
+
+add_test_pl_profile(
```

Comment:
Should these profiles be somehow added to Makefiles as well? Is that even feasible?

Reply:
Added